configs/opencv-ovms/grpc_go/stream_density_run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+/home/pipeline-server/stream_density_framework-pipelines.sh /app/entrypoint.sh
```

Comment:
Would this make sense as an environment variable in the profile?

Reply:
the value can be, but no matter what you have to have this script so that docker run can run; otherwise it fails.

Reply:
and the name of script to run is actually an environment variable inside the ovms-client